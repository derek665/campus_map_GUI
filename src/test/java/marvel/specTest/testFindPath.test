# test the findPath method is correct

LoadGraph testPath soccer.tsv

FindPath testPath Barcelona Southampton