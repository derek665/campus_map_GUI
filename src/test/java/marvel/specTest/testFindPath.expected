# test the findPath method is correct

loaded graph testPath

path from Barcelona to Southampton:
Barcelona to Valencia via La-Liga
Valencia to Arsenal via European-League
Arsenal to Southampton via EPL